### PR TITLE
Fix search providers JSON formatting in the documentations

### PR DESCRIPTION
**Pull request by MikeAlhayek**

### CODE REVIEW

File: src/docs/reference/modules/Elasticsearch/README.md [hunk 1]
```diff
@@ -55,20 +55,24 @@ Here is a sample step:
...
+                        "IndexedContentTypes":[
+                            "Article",
+                            "BlogPost"
+                        ]
+                    }
+                }
+            ]
+        }
+    ]
+}
```
Comment (Piedone) [inline src/docs/reference/modules/Elasticsearch/README.md:75]: Note the added spaces after each colon. This is how recipes are generated so the docs should reflect that too. This also applies to all the other changes.

```suggestion
    "steps": [
        {
            "name":" ElasticIndexSettings",
            "Indices": [
                {
                    "Search": {
                        "AnalyzerName": "standardanalyzer",
                        "IndexLatest": false,
                        "IndexedContentTypes": [
                            "Article",
                            "BlogPost"
                        ]
                    }
                }
            ]
        }
    ]
}
```

**Review by Piedone — CHANGES_REQUESTED** [review]
A PR title more specific like "Fix Elasticsearch JSON formatting" would be better I think.

**Comment by Skrypt** [comment]
Mike, I don't understand this Pull Request at all.

**Comment by MikeAlhayek** [comment]
> Mike, I don't understand this Pull Request at all.

@Skrypt 
just fixing the JSON formatting in the document. 

For example, the current documents show that the recipe step should look like this 
```json
{
  "name": "ElasticIndexSettings",
  "Indices": [
    {
      "Search": {
        "AnalyzerName": "standardanalyzer",
        "IndexLatest": false,
        "IndexedContentTypes": [
          "Article",
          "BlogPost"
        ]
      }
    }
  ]
},
``` 

but it should look like this instead

```json
{
    "steps":[
        {
            "name":"ElasticIndexSettings",
            "Indices":[
                {
                    "Search":{
                        "AnalyzerName":"standardanalyzer",
                        "IndexLatest":false,
                        "IndexedContentTypes":[
                            "Article",
                            "BlogPost"
                        ]
                    }
                }
            ]
        }
    ]
}
```

the current json should be in a step not at the root level.

File: src/docs/reference/modules/Elasticsearch/README.md (hunk 1 above)
Comment (MikeAlhayek) [reply src/docs/reference/modules/Elasticsearch/README.md]: @Piedone done.

**Review by Piedone — APPROVED** [review]

**Comment by hishamco** [comment]
@MikeAlhayek could you please have a look into other places, we might have the same extact issue in the entire docs

**Comment by jtkech** [comment]
Hmm I think both are okay, currently the doc is intended to describe the step itself.

At least maybe better to keep the indentation of 2 spaces that we are using in json files.

**Comment by MikeAlhayek** [comment]
@hishamco not is this PR. I noticed this as I was working on search. Maybe we look at others when I use other modules.

@jtkech changed to 2 spaces instead of 4.

**Pull request merged**